### PR TITLE
Add more service names in donet, python, js and java csv files

### PR DESCRIPTION
**Pull request by v-xuto**
1. Add more service names in donet-packages.csv, python-packages.csv , js-packages.csv and java-packages.csv.

@jongio for notification.

### CODE REVIEW

File: _data/releases/latest/dotnet-packages.csv [hunk 1]
```diff
@@ -35,7 +35,7 @@
 "Azure.Security.KeyVault.Secrets","4.1.0","4.2.0-beta.2","Key Vault - Secrets","Key Vault","keyvault","","","client","true","","Replaces: Microsoft.Azure.KeyVault"
 "Azure.AI.MetricsAdvisor","","1.0.0-beta.2","Metrics Advisor","Metrics Advisor","NA","","","client","true","",""
 "dotnet-Perks.CodeGen","2.0.1","","Perks CodeGen","Perks CodeGen","NA","NA","NA","client","true","true",""
-"Azure.Data.SchemaRegistry","","1.0.0-beta.2","Schema Registry","Schema Registry","schemaregistry","","","client","true","",""
+"Azure.Data.SchemaRegistry","","1.0.0-beta.2","Event Hubs - Schema Registry","Event Hubs","schemaregistry","","","client","true","",""
```
Comment (jongio) [inline _data/releases/latest/dotnet-packages.csv:38]: why event hubs?

File: _data/releases/latest/dotnet-packages.csv [hunk 2]
```diff
@@ -46,32 +46,32 @@
...
 "Azure.AI.TextAnalytics","5.0.0","5.1.0-beta.2","Text Analytics","Text Analytics","textanalytics","","","client","true","","Replaces: Microsoft.Azure.CognitiveServices.Language.TextAnalytics"
 "Azure.ResourceManager.AppConfiguration","","1.0.0-preview.1","Resource Management - AppConfiguration","App Configuration","appconfiguration","","","mgmt","true","",""
 "Azure.ResourceManager.Communication","","1.0.0-beta.2","Resource Management - Communication","Communication","communication","","","mgmt","true","",""
-"Azure.ResourceManager.Compute","","1.0.0-preview.2","Resource Management - Compute","Compute","compute","","","mgmt","true","",""
+"Azure.ResourceManager.Compute","","1.0.0-preview.2","Resource Management - Compute","Virtual Machines","compute","","","mgmt","true","",""
 "Azure.ResourceManager.CosmosDB","","1.0.0-preview.1","Resource Management - CosmosDB","Cosmos DB","cosmosdb","","","mgmt","true","",""
-"Azure.ResourceManager.Dns","","1.0.0-preview.1","Resource Management - DNS","DNS","dns","","","mgmt","true","",""
+"Azure.ResourceManager.Dns","","1.0.0-preview.1","Resource Management - DNS","DNS zones","dns","","","mgmt","true","",""
```
Comment (jongio) [inline _data/releases/latest/dotnet-packages.csv:60]: why zones?

File: _data/releases/latest/dotnet-packages.csv [hunk 3]
```diff
@@ -46,32 +46,32 @@
...
 "Azure.ResourceManager.EventHubs","","1.0.0-preview.1","Resource Management - EventHubs","Event Hubs","eventhub","","","mgmt","true","",""
 "Azure.ResourceManager.Insights","","1.0.0-preview.1","Resource Management - Insights","Insights","insights","","","mgmt","true","",""
 "Azure.ResourceManager.KeyVault","","1.0.0-preview.1","Resource Management - KeyVault","Key Vault","keyvault","","","mgmt","true","",""
-"Azure.ResourceManager.Network","","1.0.0-preview.2","Resource Management - Network","Network","network","","","mgmt","true","",""
+"Azure.ResourceManager.Network","","1.0.0-preview.2","Resource Management - Network","Virtual Network","network","","","mgmt","true","",""
```
Comment (jongio) [inline _data/releases/latest/dotnet-packages.csv:64]: Why virtual?

File: _data/releases/latest/dotnet-packages.csv [hunk 4]
```diff
@@ -620,91 +620,91 @@
...
-"Microsoft.Azure.Management.HDInsight","6.0.0","","Management - HD Insight","HD Insight","hdinsight","https://docs.microsoft.com/dotnet/api/overview/azure/hdinsight","NA","mgmt","false","",""
-"Microsoft.Azure.Management.HDInsight.Job","2.0.7","","Management - HD Insight Job","HD Insight","NA","https://docs.microsoft.com/dotnet/api/overview/azure/hdinsight","NA","mgmt","false","",""
-"Microsoft.Azure.Management.HealthcareApis","3.0.0","","Management - Health Care APIs","Health Care APIs","healthcareapis","NA","NA","mgmt","false","",""
+"Microsoft.Azure.Management.FrontDoor","3.0.0","","Management - Front Door","Azure Front Door","frontdoor","NA","NA","mgmt","false","",""
+"Microsoft.Azure.Management.Graph.RBAC.Fluent","1.35.0","","Management - Graph RBAC Fluent","Microsoft Graph","NA","NA","NA","mgmt","false","",""
+"Microsoft.Azure.Management.GuestConfiguration","1.4.0","","Management - Guest Configuration","Resource Explorer","guestconfiguration","NA","NA","mgmt","false","",""
+"Microsoft.Azure.Management.HDInsight","6.0.0","","Management - HdInsight","HdInsight","hdinsight","https://docs.microsoft.com/dotnet/api/overview/azure/hdinsight","NA","mgmt","false","",""
```
Comment (jongio) [inline _data/releases/latest/dotnet-packages.csv:647]: HD

**Review by jongio — CHANGES_REQUESTED** [review]

File: _data/releases/latest/dotnet-packages.csv (hunk 4 above)
Comment (v-xuto) [reply _data/releases/latest/dotnet-packages.csv]: I have updated it.

File: _data/releases/latest/dotnet-packages.csv (hunk 2 above)
Comment (v-xuto) [reply _data/releases/latest/dotnet-packages.csv]: I have updated it.

File: _data/releases/latest/dotnet-packages.csv (hunk 3 above)
Comment (v-xuto) [reply _data/releases/latest/dotnet-packages.csv:64]: Refer this link: https://docs.microsoft.com/en-us/dotnet/api/overview/azure/virtual-network?view=azure-dotnet.

File: _data/releases/latest/dotnet-packages.csv (hunk 1 above)
Comment (v-xuto) [reply _data/releases/latest/dotnet-packages.csv:38]: Refer this link: https://docs.microsoft.com/en-us/azure/event-hubs/schema-registry-overview

**Comment by v-xuto** [comment]
@jongio - We have updated the csv files about four languages (donet, java, js and python). Please Review.

**Review by jongio — CHANGES_REQUESTED** [review]

**Comment by jongio** [comment]
@v-xuto - Please retarget this to azure-sdk:master instead of jongio:more-services  Thanks

**Pull request closed without merging**